.neuro/live-pachyderm.yml

```diff
@@ -165,31 +113,26 @@ jobs:
     http_port: 443
     http_auth: False
     life_span: 1d
+    pass_config: True
     detach: False
     browse: True
     volumes:
       - secret:gh-rsa:/root/.ssh/id-rsa
     env:
       PROJECT: /usr/project
+      DATA_PATH: /usr/project/data/Images
       LABEL_STUDIO_LOCAL_FILES_SERVING_ENABLED: True
       LABEL_STUDIO_PROJECT: /label-studio-project
-      PACHY_TOKEN: secret:pachyderm-token
       LS_TOKEN: secret:ls-token
       LS_PORT: 443
     bash: |
-      echo "Downloading base dataset"
-      echo "IdentityFile ~/.ssh/id-rsa" > ~/.ssh/config
-      ssh-keyscan github.com >> ~/.ssh/known_hosts
       git clone -b ${PROJECT_GIT_BRANCH} ${PROJECT_GIT_REPO} ${PROJECT}
 
-      # Init pachctl
+      # Init pachctl and pull dataset
       pachctl config update context default --pachd-address ${PACHY_URI}
-      echo "${PACHY_TOKEN}" | pachctl auth use-auth-token
+      pachctl get file ${PACHY_REPO}@master:/data/ -r -o ${PROJECT}/data/ | tee
 
-      export DATA_PATH=${PROJECT}/data/Images
       COMPLETIONS=${LABEL_STUDIO_PROJECT}/completions
```

Comment:
this is not actual anymore, let's remote it (used in commit message for pachyderm)